puan/modules/configurator/__init__.py

```diff
@@ -321,14 +321,16 @@ def select(self, *prios: typing.List[typing.Dict[str, int]], solver: typing.Call
                     self.leafs()
                 )
             )
-            res = itertools.starmap(
-                lambda config,ov,sc: dict(
-                    filter(
-                        lambda x: x[0] in leafs,
-                        config.items()
-                    )
-                ),
-                res
+            res = list(
+                itertools.starmap(
+                    lambda config,ov,sc: dict(
+                        filter(
+                            lambda x: x[0] in leafs,
+                            config.items()
+                        )
+                    ),
+                    res
+                )
```

Comment:
Return iterator instead. Please double check that `ndarray.ge_polyhedron_config.solve` and `plog.AtLeast.solve` also returns an iterator

Reply:
Fixed, had to update the methods you mentioned. Also updated the configurator example in the quickstart guide